chall04/alcohen.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python3
+
+# Version: Python 3.6.9
+
+def error_input(msg):
+    print(msg)
+    quit()
+
+def can_move(next_under):
+    if (next_under == ' '):
+        return True
+    return False
+
+if __name__ == '__main__':
```

Comment:
I would recommend putting the whole block inside a function main. But it's really good you protected it.